src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/SubaccountProcessor.kt

```diff
@@ -107,9 +110,16 @@ internal open class SubaccountProcessor(
         state = process(
             existing = state,
             payload = subaccount,
-            firstTime = true,
+            firstTime = subscribed,
```

Comment:
[q] does this mean after the first time processing, we will be setting the `subscribed` bool to false?

Reply:
Right... the same process call is used for both cases.

Reply:
wait, the same process call is used for both cases? both cases as in the first time and second time? not 100% sure what you mean in terms of that

Reply:
Yes, but the first time and the second time.